.github/actions/javascript/createOrUpdateStagingDeploy/createOrUpdateStagingDeploy.js

```diff
@@ -122,13 +122,20 @@ const run = function () {
                 'number',
             );
 
+            // Get the internalQA PR list, preserving the previous state of `isResolved`
+            const internalQAPRList = _.sortBy(
+                currentStagingDeployCashData.internalQAPRList,
+                'number',
+            );
+
             return GithubUtils.generateStagingDeployCashBody(
                 newTag,
                 _.pluck(PRList, 'url'),
                 _.pluck(_.where(PRList, {isVerified: true}), 'url'),
                 _.pluck(_.where(PRList, {isAccessible: true}), 'url'),
                 _.pluck(deployBlockers, 'url'),
                 _.pluck(_.where(deployBlockers, {isResolved: true}), 'url'),
+                _.pluck(_.where(internalQAPRList, {isResolved: true}), 'url'),
                 didVersionChange ? false : currentStagingDeployCashData.isTimingDashboardChecked,
                 didVersionChange ? false : currentStagingDeployCashData.isFirebaseChecked,
```

Comment:
And then passing as a parameter an array of URLs of only those PRs which are resolved (the checkbox has been ticked)